src/shell-interface.h

```diff
@@ -21,8 +21,11 @@
 #ifndef wasm_shell_interface_h
 #define wasm_shell_interface_h
 
-#include <wasm.h>
-#include <wasm-interpreter.h>
+#include "shared-constants.h"
+#include "asmjs/shared-constants.h"
+#include "support/name.h"
+#include "wasm.h"
+#include "wasm-interpreter.h"
```

Comment:
I don't think the includes need to be changed.

Reply:
yeah i just copied this from the latest binaryen version and removed the final keyword :^)